### PR TITLE
add user show page

### PR DESCRIPTION
**Pull request by casasmdavid**
added user show page and rspec for it.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -3,4 +3,15 @@ class UsersController < ApplicationController
   def index
     @users = User.all
   end
+
+  def show
+    @user = User.find(params[:id])
+
+    if @user.profile_photo.model.profile_photo_url.nil?
+      @profile_photo =
+        'https://www.idyllwildarts.org/wp-content/uploads/2016/09/blank-profile-picture.jpg'
```
Comment (jdmacmurtrie) [inline app/controllers/users_controller.rb]: Lets abstract this logic into a private method.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -3,4 +3,15 @@ class UsersController < ApplicationController
...
+    else
+      @profile_photo = @user.profile_photo
+    end
```
Comment (jdmacmurtrie) [inline app/controllers/users_controller.rb:30]: Let's abstract this logic into a private method down below.

File: spec/features/user_views_profile_spec.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  So that I can see my account information) do
+  scenario 'view account information' do
+    user = FactoryBot.create(:user)
+
+    visit user_path(user)
+
+    expect(page).to have_content(user.username)
+    expect(page).to have_content(user.email)
+    expect(page).to have_content(user.profile_photo)
+  end
```
Comment (jdmacmurtrie) [inline spec/features/user_views_profile_spec.rb:16]: What if a user tries to log into a profile page that's not theirs?

File: spec/features/user_views_profile_spec.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+feature 'user checks his profile page', %(
```
Comment (jdmacmurtrie) [inline spec/features/user_views_profile_spec.rb]: Gender neutral please.

File: spec/features/user_views_profile_spec.rb [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  As a signed up user
+  I want to view my profile page
+  So that I can see my account information) do
+  scenario 'view account information' do
+    user = FactoryBot.create(:user)
```
Comment (jdmacmurtrie) [inline spec/features/user_views_profile_spec.rb]: Let's use the `let` syntax for FactoryBot.

File: spec/features/user_views_profile_spec.rb (hunk 3 above)
Comment (casasmdavid) [reply spec/features/user_views_profile_spec.rb:16]: we plan to set this up once we have added admin functionality

**Pull request merged**